artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/queues.js

```diff
@@ -125,18 +125,23 @@ var Artemis;
         };
         ctrl.tableColumns = [
             { header: 'ID', itemField: 'id' },
-            { header: 'Name', itemField: 'name' },
+            { header: 'Name', itemField: 'name',
+              templateFn: function(value, item) { return '<a href="#" onclick="selectQueue(\'' + item.address + '\',\'' + value + '\',\'' + getQueuesNid(item, $location) + '\')">' + value + '</a>' }
+            },
             { header: 'Routing Types', itemField: 'routingTypes' },
-            { header: 'Queue Count', itemField: 'queueCount' },
```

Comment:
Was this accidental?

Reply:
This was on purpose, the queue object has no queueCount property.